syntax = "proto3";

package protocols.common;

enum MessageType {
  MESSAGE_TYPE_UNSPECIFIED = 0;
  MESSAGE_TYPE_SSL_SPEED = 1;
  MESSAGE_TYPE_SSL_POSITION = 2;
  MESSAGE_TYPE_SSL_TELEMETRY = 3;
  MESSAGE_TYPE_SSL_ODOMETRY = 4;
  MESSAGE_TYPE_SSL_MOTORS = 5;
}
